### ISSUE TITLE
Is there any plans to mirror H2-xxxx?

### ISSUE THREAD

**Issue by Kondybas** [bigtreetech/H2-Series#8]
I have installed H2-V2S on Ender3 and I have a consideration on extruder design.

To save as much printing area as possible I had to move extruder far to the right and even then I have lost about 10mm of the X. 
Two topmost holes on the creality carriage is the best possible place to mount H2-xxxx

![processed-095740c8-a44c-4751-a433-2d2839570d5d_hhXojsBA](https://github.com/bigtreetech/H2-Series/assets/75861332/87ad4f92-1684-420c-beac-a4825b5568cc)

Now the motor is hanging on the right far away from the carriage center of gravity. I beleive that would be much better simply to mirror the extruder - with motor at left and fan at right. That would make the whole setup much easier and also would make a better mass distribution.

**Comment by Kondybas** [comment]
Here is the photo depicting H2-V2S mounted on a creality X-carriage. 

![1703948085194](https://github.com/bigtreetech/H2-Series/assets/75861332/fecf8dba-7cbb-4286-b062-62ccdead91e6)

**Comment by ZaLiTHkA** [comment]
hey @Kondybas, just the other day I found [a video on YouTube](https://www.youtube.com/watch?v=Hlu4NTe6Mi0) where I saw the H2 V2S mounted the other way around, just like you described.

take note: this video doesn't explicitly cover *how* it was mounted or *why* it was done that way, he does mention having another video regarding that, but unfortunately I couldn't find it.. the video description has a Thingiverse link to the model files.

I'm only just getting started on my 3D printing adventure, so I don't think I can really help much more than that right now.. 🍻 with that said, I have an Ender 3 v2 (with the stock hotend and bowden extruder) and a H2 V2S (non-Lite version) sitting in a box while I research mounting options, so hopefully I'll know more soon. 😁

**Comment by Kondybas** [comment]
Hi Andre, at 2:43 a printed bracket shown which is used for H2 mounting. This approach makes unaccessible the feeder wheel that is used for nozzle cleaning (heating under pull tension). 

Speaking frankly I have used the stock E3Pro since 2019 for a year with almost no problems. Then I replaced mainboard for quiet drivers (SKR E3-mini-DIP) and used it for another 3 years. I had pretty good prints of PETG with bowden and the only reason I tried direct is a flexible plastics. And H2 works exactly as it was expected - very well. 

H2 is very good extruder, its only minus on Ender is that some printable area should be sacrificed. Or - as shown on video you have found - feeder wheel accessibility. All that can be easily avoided with the mirrored H2 - for Ender and all its clones.

**Comment by ZaLiTHkA** [comment]
> Hi Andre, at 2:43 a printed bracket shown which is used for H2 mounting. This approach makes unaccessible the feeder wheel that is used for nozzle cleaning (heating under pull tension).

yeah, that is one downside of mounting the H2 the other way around..

> H2 is very good extruder, its only minus on Ender is that some printable area should be sacrificed. Or - as shown on video you have found - feeder wheel accessibility. **All that can be easily avoided with the mirrored H2** - for Ender and all its clones.

..but I suspect I may have misunderstood this part. I thought you were simply looking for other mounting options, but it seems you're actually suggesting a "mirrored" H2 design, is that correct? as in, another H2 model that would end up with the filament feed wheel on the other side?

**Comment by Kondybas** [comment]
Here is the blueprints of H2 and E3 X-carriage combined. Upper is current mount, lower - for mirrored H2
![BTTH2-Mirror](https://github.com/user-attachments/assets/19a39163-4d91-4ee0-a74e-c6c7cbb59632)

As you can see on the upper scheme even with H2 moved to the far right its nozzle isn't aligned with  original nozzle position. H2 could be moved even more, with some bracket, but then its motor will hit the right carriage of the Z axis. 

I hope it is quite obvious that lower scheme with mirrored H2 have no issues at all.

**Comment by ZaLiTHkA** [comment]
gotcha, that makes perfect sense. even though I haven't used my H2 V2S yet, the "standard way" of how to mount it on my Ender 3 v2 bracket was what sent me down the rabbithole that led me to this GitHub post.

to be fair, I have no idea how many different things the designers had to consider while making this. so I'm not sure if you could say with full certainty that...

> I hope it is quite obvious that lower scheme with mirrored H2 have no issues at all.

having said that, I do agree that the current standard mounting idea is far from optimal.

---

just a thought: I wonder if this is actually the right place to pose this question.. I mean, do "the right people", as in those who could make decisions like this, actually come look at GitHub as well?

**Comment by Kondybas** [comment]
> I have no idea how many different things the designers had to consider

Well, I made at least five major changes to the first design and I had ruined one X-carriage for R&D. I bought two used carriages with loose hotend stubs (a common problem) - one for testing and one for production. And I had to grind off bolt heads with a 0.1mm precision to fit the gaps. Sure, it was a lot of fun but I would like to have a mirrored H2, it would be much less painful.

**Comment by Kondybas** [comment]
>  I wonder if this is actually the right place to pose this question.. I mean, do "the right people", as in those who could make decisions like this, actually come look at GitHub as well?

This repo is owned by BTT and I hope someone have visit it time after time. 

I summon the spirit of the developer in the chat!

**Comment by bigtreetech** [comment]
You summoned me?

**Comment by Kondybas** [comment]
Obey and answer: will there be a mirror version?
